src/Stormpath/DataStore/DefaultDataStore.php

```diff
@@ -291,6 +291,11 @@ private function toStdClass(Resource $resource, $customData = false)
                 $property = $this->toSimpleReference($name, $property);
             }
 
+            else if ($property instanceof \Stormpath\Resource\Resource)
+            {
+                $property = $this->toStdClass($property);
+            }
+
```

Comment:
The problem was that the `accountStore` property wasn't getting included in the request to the server. Without this case the resulting `\stdClass` class object would contain an `accountStore` property with an empty value.